frontend/app/components/Spots/SpotPlayer/components/SpotPlayerHeader.tsx

```diff
@@ -55,6 +58,18 @@ function SpotPlayerHeader({
     history.push(spotLink);
   };
 
+  // Create menu for dropdown
+  const menu = (
```

Comment:
depreceted api, use `menu={{ items: menuItems, onClick: onMenuClick }}` prop for dropdown instead